pkg/cmd/step/step_tag.go

```diff
@@ -94,7 +95,7 @@ func NewCmdStepTag(commonOpts *opts.CommonOptions) *cobra.Command {
 	cmd.Flags().StringVarP(&options.Flags.ChartValueRepository, "charts-value-repository", "r", "", "the fully qualified image name without the version tag. e.g. 'dockerregistry/myorg/myapp'")
 
 	cmd.Flags().BoolVarP(&options.Flags.NoApply, "no-apply", "", false, "Do not push the tag to the server, this is used for example in dry runs")
-
+	cmd.Flags().BoolVarP(&options.Flags.ForcePush, "force-push", "", false, "force push the tag to the server")
```

Comment:
I am a bit concerned around force pushing tags. Where do you want to use this? What if someone else already has received the old tags? Will he not get errors/warnings when pulling the next time?

Reply:
What is the underlying problem we are trying to solve?